src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -121,6 +121,8 @@ void UpdateClip()
 			var geometricClip = compositor.CreateGeometricClip(pathGeometry);
 
 			visual.Clip = geometricClip;
+			//When the clip is updated, the shadow must be updated as well
+			UpdateShadowAsync().FireAndForget(IPlatformApplication.Current?.Services?.CreateLogger(nameof(WrapperView)));
```

Comment:
This works if the View already have a Shadow, but, works if the order is:
1. Clip
2. Add a Shadow
?

Reply:
@jsuarezruiz , A new shadow will be created if the order is reversed.